client/ayon_core/hosts/substancepainter/api/pipeline.py

```diff
@@ -76,7 +74,7 @@ def install(self):
         log.info("Installing menu ... ")
         self._install_menu()
 
-        project_settings = get_current_project_settings()
+        project_settings = get_project_settings(get_current_project_name())
```

Comment:
Is `get_current_project_settings` deprecated?

Reply:
> Is get_current_project_settings deprecated? Is there a reason for this code change?

I had some local changes and didn't realized there was `get_current_project_settings` before. But it is good question. Function `get_current_project_settings` is using envrionment variable for current project, but that may be different from `get_current_project_name`...